server/api/core/email/email.js

```diff
@@ -27,27 +26,10 @@ export function send(options) {
  * @returns {Object} returns source
  */
 export function getTemplate(template) {
-  check(template, String);
```

Comment:
Sorry, I wasn't aware that was being recommended.  I searched the whole app to see if there was any API for using the Templates collection (other than the standard `Mongo` methods) and found nothing, so I thought that was just a placeholder.  And #1362 didn't exist when I changed that code.  

More than anything else, I was just trying to improve performance by removing the database queries that I didn't think were doing anything yet.  But also, the language result from `Meteor.call("shop/getLocale")` was being set to a global variable and then never used.